src/components/bikes.js

```diff
@@ -0,0 +1,20 @@
+import './bikes.scss';
+import { createElement } from '../lib/dom';
+
+export function createSearchResults(props) {
+  const container = createElement('div', {
+    className: 'bikes'
+  });
+
+  props.forEach(item => {
```

Comment:
`props` sounds like an object, containing multiple properties.
Actually, your only argument is `filteredBikes`. 
I would change the call to:
```
createSearchResults({
  items: filteredBikes,
  onSearchResultClick: .... 
})
```
Right now, your event listener will throw an error (`onSearchResultClick is not a function`).

Reply:
Good point. have adapted this and will add the event listener in the next step.